client/helpers.go

```diff
@@ -108,3 +111,60 @@ func WaitForTxConfirmationAndFee(ctx context.Context, ethClient *ethclient.Clien
 	}
 	return new(big.Int).Mul(tx.GasPrice(), big.NewInt(0).SetUint64(r.GasUsed)), nil
 }
+
+func ParseSigs(sigs []byte, hash []byte, validators []common.Address) ([]uint8, [][32]byte, [][32]byte, []*big.Int, error) {
```

Comment:
lets add a test from this function, and some comments around what it actually does

Reply:
Just added a test to its loom-js counterpart, I'll do that today.